svgelements/svgelements.py

```diff
@@ -43,7 +43,7 @@
 and the Arc can do exact arc calculations if scipy is installed.
 """
 
-SVGELEMENTS_VERSION = "1.5.5"
+SVGELEMENTS_VERSION = "1.5.6"
```

Comment:
Changing preserve_aspect_ratio to preserveAspectRatio would be a breaking change and the version would go to 1.6.0

Reply:
Going to revert breaking changes.